File: Programmers/월간코드챌린지/Lv1_없는숫자더하기.java
import java.util.*;

class Solution {
    public int solution(int[] numbers) {
        int answer = 0;
        boolean visited[] = new boolean[10];
        for (int number : numbers) {
            visited[number] = true;
        }
        for (int i = 0; i < 10; i++) {
            if (visited[i] == false) {
                answer += i;
            }
        }
        return answer;
    }
}

File: Programmers/월간코드챌린지/Lv1없는숫자더하기.kt
class Solution {
    fun solution(numbers: IntArray): Int {
        val intArray = IntArray(10)
        for (number: Int in numbers) {
            intArray[number] = 1
        }
        return (0 until 10).asSequence()
            .filter { intArray[it] == 0 }
            .sum()
    }
}